lib/sis_scraper.rb

```diff
@@ -0,0 +1,159 @@
+require 'sis_scraper.rb'
+
+class SISScraper
+  ALPHA = ('A'..'Z').to_a
+
+  DOWNLOAD_DIR = '../../../Downloads/'
+
+  def initialize()
```

Comment:
You missed the download_dir as an argument, even though you're using it in your rake task.